cmd/qri.go

```diff
@@ -20,14 +20,14 @@ func NewQriCommand(ctx context.Context, pf PathFactory, generator gen.CryptoGene
 	cmd := &cobra.Command{
 		Use:   "qri",
 		Short: "qri GDVCS CLI",
-		Long: `qri ("query") is a global dataset version control system 
-on the distributed web.
-
-https://qri.io
-
-Feedback, questions, bug reports, and contributions are welcome!
+		Long: `qri ("query") is a set of tools for building & sharing datasets: https://qri.io
+Feedback, questions, bug reports, and contributions are welcome! 
 https://github.com/qri-io/qri/issues`,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if ioStreams.IsCygwinTerminal() {
```

Comment:
This now superseeds my PR https://github.com/qri-io/qri/pull/1263 will close that out.

Reply:
Though think we are both wrong:
https://github.com/fatih/color/pull/61#pullrequestreview-22449122
+ I think we should look into why `fatih/color` doesn't do this correctly, `v1.7.0` should handle it well, without us actually setting this.

Reply:
innnfteresting. looks like the latest release of `color` is `1.9.0`. We could try upgrading. I can revert this change until we figure it out?

Reply:
Sounds good, we can keep the original issue continue tracking this.